capture-sdk/sdk/src/main/res/drawable/gc_only_qr_code_scanning_frame.xml

```diff
@@ -0,0 +1,26 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
[Clarification] Where is this used? When I run the example app, then it shows the normal frame.

Reply:
Oh yeah, that can be deleted it's not needed anymore. Will delete that.

Reply:
But the design shows a square for QR code only mode. So we do need it. Let's talk about it tomorrow.

Reply:
[Scan](https://www.figma.com/file/LJT8L5kpqe00Zbb2NDK55B/Gini-Photo-Payment-%2F-Android?node-id=2458%3A11788)